assets/sass/scaffolding/scope-prose.scss

```diff
@@ -265,15 +265,14 @@ $constrainedWide: 42em;
             margin: 1em 0 2em;
         }
 
-        &.auto-width {
-            img {
-                width: auto;
-            }
+        img {
```

Comment:
Need to test this a bit more, but I _think_ we're OK to make all images/figures in content areas have an auto width rather than relying on the `auto-width` class.